src/color.rs

```diff
@@ -1072,3 +1240,55 @@ where
 
     Ok(into_color(lightness, chroma, hue, alpha))
 }
+
+#[inline]
+fn parse_color_function<'i, 't, ComponentParser>(
+    component_parser: &ComponentParser,
+    arguments: &mut Parser<'i, 't>,
+) -> Result<Color, ParseError<'i, ComponentParser::Error>>
+where
+    ComponentParser: ColorComponentParser<'i>,
+{
+    let location = arguments.current_source_location();
+    let color_space = arguments.try_parse(|i| {
+        let ident = i.expect_ident()?;
+        Ok(match_ignore_ascii_case! {
+            ident,
+            "srgb" => PredefinedColorSpace::Srgb,
+            "srgb-linear" => PredefinedColorSpace::SrgbLinear,
+            "display-p3" => PredefinedColorSpace::DisplayP3,
+            "a98-rgb" => PredefinedColorSpace::A98Rgb,
+            "prophoto-rgb" => PredefinedColorSpace::ProphotoRgb,
+            "rec2020" => PredefinedColorSpace::Rec2020,
+            "xyz-d50" => PredefinedColorSpace::XyzD50,
+            "xyz" | "xyz-d65" => PredefinedColorSpace::XyzD65,
```

Comment:
Let's use `from_str`, or remove `from_str` once it's not used for serialization?